repos/system_upgrade/el7toel8/actors/enablecrbrepo/libraries/library.py

```diff
@@ -0,0 +1,22 @@
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import UsedTargetRepositories
+
+
+CRB_REPOID = 'codeready-builder-for-rhel-8-x86_64-rpms'
```

Comment:
What about moving this to config actor?

Reply:
Nope as this is not anything related to the config